documentation/book/proc-upgrading-brokers-newer-kafka.adoc

```diff
@@ -6,7 +6,7 @@
 
 = Upgrading Kafka brokers and client applications
 
-This procedure describes how you can upgrade a {ProductName} Kafka cluster to a higher version of Kafka, such as upgrading from version {KafkaVersionLower} to {KafkaVersionHigher}.
+This procedure describes how you can upgrade a {ProductName} Kafka cluster to a higher version of Kafka.
```

Comment:
@PaulRMellor suggested edit: "This procedure describes how to upgrade a {ProductNam}..."